packages/commons/README.md

```diff
@@ -21,6 +21,30 @@ This package exists primarily to help .NET tool developers integrate `axe-core`
 * [Deque.AxeCore.Playwright](../playwright/README.md) in combination with [Playwright for .NET](https://playwright.dev/dotnet/)
 * [Deque.AxeCore.Selenium](../selenium/README.md) in combination with [Selenium](https://www.selenium.dev/)'s [C# Selenium.WebDriver package](https://www.nuget.org/packages/Selenium.WebDriver)
 
+
+## `CachedContentDownloader`
+
+Allows cacheing of downloaded external resources.
+
+## `ContentDownloader`
+
+Provides access to content from specific resources using a provided URL.
+## `EmbeddedResourceAxeProvider`
+
+This class is used to provide access to an embedded axe resources for accessibility.
+
+## `EmbeddedResourceProvider`
```

Comment:
`EmbeddedResourceProvider` should also not be documented, it's an internal implementation detail of `EmbeddedResourceAxeProvider`

Reply:
Removed!